docs/content/latest/troubleshoot/cluster/replace_failed_peers.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Fix 2/3 failed peers
+linkTitle: Fix 2/3 failed peers
+description: Fix 2/3 failed peers
+aliases:
+  - /troubleshoot/cluster/fix-2-3-failed-peers/
+  - /latest/troubleshoot/cluster/fix-2-3-failed-peers/
+menu:
+  latest:
+    parent: troubleshoot-cluster
+    weight: 835
+isTocNested: true
+showAsideToc: true
+---
+
+When a RAFT peer fails, YugabyteDB executes an automatic remote bootstrap to create a new peer from the remaining ones.
+If 2/3 of RAFT peers fail for a given tablet, we have to manually execute the equivalent of a remote bootstrap. We
+ can get a list of tablets in `yb-master-ip:7000/tablet-replication` yb-admin gui. 
+
+
+Assuming we have a cluster where:
```

Comment:
And here would could also mention that we're assuming RF3 for this example

Reply:
Mentioned.